Makefile

```diff
@@ -68,6 +68,9 @@ build-push-all: container-build-operator container-push-operator container-build
 upgrade-test:
 	./hack/upgrade-test.sh
  
+check-state:
+	timeout 10m bash -c 'export CMD="./cluster-up/kubectl.sh";exec ./hack/check-state.sh'
```

Comment:
./cluster-up/kubectl.sh will fail on Prow since it only works with STDCI.

Reply:
removed these lines. now it is no longer hard coded.